src/Core/Output/GlobalDataWriter.cs

```diff
@@ -205,19 +205,17 @@ public bool VisitArray(ArrayType at)
             fmt.Terminate();
             fmt.Indent();
             fmt.Write("{");
-            fmt.Terminate();
-            fmt.Indentation += fmt.TabSize;
+            fmt.Write(" ");
```

Comment:
You've changed the rendering of arrays so that they are rendered on a single line. What happens when an array of 1000 entries is encountered?  I prefer the current state of affairs, until Reko can prettify the output source.

In addition, you haven't changed the `GlobalDataWriterTests.cs` unit tests to match, so the CI pipeline will fail. Make sure there is a unit tests to cover this case.

There is the beginning of a code prettifier in `Core\Output\PrettyPrinter.cs` that should be able to format arrays nicely. Consider opening an issue and using that class to make data output prettier.